site/client/runCountryProfilePage.ts

```diff
@@ -117,9 +118,8 @@ class ChartFilter {
                 c.li.style.display = "none"
             } else {
                 c.li.style.display = ""
-                c.li.children[0].children[0].innerHTML = fuzzysort.highlight(
-                    res
-                )
+                c.li.children[0].children[0].innerHTML =
+                    fuzzyHighlight(res) ?? ""
```

Comment:
Quick question: so previously `innerHTML = fuzzysort.highlight( res)` could be getting set to null, and now you've changed it to it to "" if fuzzyHiglight returns null?

Reply:
That's a type question just there, since `fuzzyHighlight` can return `null`, but `innerHtml` cannot be set to `null` (I don't think this could happen in this case, but TS dislikes it anyway).

This is the new "nullish coalescing" operator in TS or ES2020, which basically evaluates to `fuzzyHighlight(res) != null ? fuzzyHighlight(res) : ""`.
It's similar to a simple `||`, but wouldn't act on a "false-ish" value, only on a nullish one.